easy-random-core/src/main/java/org/jeasy/random/util/CollectionUtils.java

```diff
@@ -1,58 +0,0 @@
-/*
- * The MIT License
- *
- *   Copyright (c) 2020, Mahmoud Ben Hassine (mahmoud.benhassine@icloud.com)
- *
- *   Permission is hereby granted, free of charge, to any person obtaining a copy
- *   of this software and associated documentation files (the "Software"), to deal
- *   in the Software without restriction, including without limitation the rights
- *   to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- *   copies of the Software, and to permit persons to whom the Software is
- *   furnished to do so, subject to the following conditions:
- *
- *   The above copyright notice and this permission notice shall be included in
- *   all copies or substantial portions of the Software.
- *
- *   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- *   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- *   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- *   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- *   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- *   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- *   THE SOFTWARE.
- */
-package org.jeasy.random.util;
-
-import java.util.List;
-import java.util.Random;
-
-/**
- * Collection utility methods.
- * 
- * <strong>This class is intended for internal use only.</strong>
- *
- * @author Mahmoud Ben Hassine (mahmoud.benhassine@icloud.com)
- */
-public final class CollectionUtils {
-
-    private CollectionUtils() {
-    }
-
-    /**
-     * Get a random element from the list.
-     *
-     * @param list the input list
-     * @param <T>  the type of elements in the list
-     * @return a random element from the list or null if the list is empty
-     */
-    public static <T> T randomElementOf(final List<T> list) {
-        if (list.isEmpty()) {
-            return null;
-        }
-        return list.get(nextInt(0, list.size()));
-    }
-
-    private static int nextInt(int startInclusive, int endExclusive) {
```

Comment:
This is actually duplicated in `RandomizationContext`. Both methods seems to be called always with `startInclusive = 0`, which mean this is no different than `Random.nextInt(bound)`, so we can get rid of these methods and use the built-in method instead (I will show how in other comments).